ansible/roles/pre_update_node/tasks/main.yml

```diff
@@ -1,16 +1,16 @@
 ---
-- name: Choose a single master to run commands on
-  set_fact:
-    cluster_master: "{{ groups['oo_hosttype_master'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
-  run_once: true
-  when: "{{ update_needed == True }}" 
-
-# FIXME: use the oadm_manage_node module instead
-- name: Make node unschedulable
-  command: "oadm manage-node {{ hostvars[oo_name].ec2_private_dns_name }} --schedulable=false"
-  register: unschedulable
-  until: unschedulable.stdout.find("SchedulingDisabled") != -1
-  retries: 10
-  delegate_to: "{{ cluster_master }}"
-  ignore_errors: True
-  when: "{{ update_needed == True }}" 
+#- name: Choose a single master to run commands on
+#  set_fact:
+#    cluster_master: "{{ groups['oo_hosttype_master'] | intersect(groups['oo_clusterid_' ~ oo_clusterid]) | first }}"
+#  run_once: true
+#  when: "{{ update_needed == True }}" 
+#
+## FIXME: use the oadm_manage_node module instead
+#- name: Make node unschedulable
+#  command: "oadm manage-node {{ hostvars[oo_name].ec2_private_dns_name }} --schedulable=false"
+#  register: unschedulable
+#  until: unschedulable.stdout.find("SchedulingDisabled") != -1
+#  retries: 10
+#  delegate_to: "{{ cluster_master }}"
+#  ignore_errors: True
+#  when: "{{ update_needed == True }}" 
```

Comment:
Let me be a bit more clear.

In the above post_scale.yml playbook I was shoing an examle of how to use oadm_manage_node

```
 - name: oadm manage-node --schedulable=true --selector=ops_node=new
    oadm_manage_node:
      selector: ops_node=new
      schedulable: True
```

In this case, we used a "selector" option.  I am suggesting we don't use a selector, but a "node".  This will duplicate the functionality you have above.  This module is also idempotent